### DISCUSSION TITLE
What font provides xyz?

### DISCUSSION THREAD

**Discussion by rickswe** [ryanoasis/nerd-fonts#848] [Q&A]
Is there a way to find out which font provides f241 (full battery)? Other than manually
copying every font into ~/.local/share/fonts, fc-cache, startx. Process of elimination.

My problem is that a while back I copied all the nerd fonts and started using f240 in
xmobar. Now I want to use a minimal amount of fonts but I don't know what
font uses that. My f241 battery now looks chopped in half. I have tried gnome
font manager with no success.

My temp solution is to use a standing battery icon f240 (full battery) with the
minimal fonts that I have now.

Thanks

**Comment by Finii** [comment]
> Is there a way to find out which font provides f241 (full battery)?

```
$ fc-list ':charset=f241' family
CaskaydiaCove NF
CaskaydiaCove Nerd Font Mono
```

Of course the result will be different on your machine.
The command lists all fonts that can provide a `F241` glyph (of the currently installed fonts).
